package org.sinerji.exception;

public class DataInvalidaException extends RuntimeException{

    public DataInvalidaException() {
        super("ERRO: A data buscada Ã© invalida. \n Tente novamente.");
    }

}
